code/modules/spells/spell_types/shadow_walk.dm

```diff
@@ -0,0 +1,91 @@
+/obj/effect/proc_holder/spell/targeted/shadowwalk
+	name = "Shadow Walk"
+	desc = "Grants unlimited movement in darkness."
+	charge_max = 0
+	clothes_req = 0
+	phase_allowed = 1
+	selection_type = "range"
+	range = -1
+	include_user = 1
+	cooldown_min = 0
+	overlay = null
+	action_icon = 'icons/mob/actions/actions_minor_antag.dmi'
+	action_icon_state = "bloodcrawl"
```

Comment:
Any suggestions for what icon to use?

Reply:
![shadowwalk](https://user-images.githubusercontent.com/6209658/29738241-14aafc44-89ec-11e7-9bf5-a43b8863977c.png)

try this for size